NextcloudTalk/NCCallController.m

```diff
@@ -701,7 +701,20 @@ - (void)externalSignalingController:(NCExternalSignalingController *)externalSig
 - (void)externalSignalingController:(NCExternalSignalingController *)externalSignalingController didReceivedParticipantListMessage:(NSDictionary *)participantListMessageDict
 {
     NSLog(@"External participants message received: %@", participantListMessageDict);
+    
     NSArray *usersInRoom = [participantListMessageDict objectForKey:@"users"];
```

Comment:
this does not exist in that case btw
So not sure if you need to do
```
if ([[participantListMessageDict allKeys] containsObject:@"users"]) {
```

Reply:
that check could be added but not really needed.